R/generate_price_data.R

```diff
@@ -37,82 +37,38 @@ get_prices <- function(aScenType) {
 
 #' get_hindcast_prices
 #'
-#' @details Read in FAO prices by country and crop,
-#'          Aggregate to GCAM regions and commodities,
+#' @details Read in FAO prices by GCAM regions and commodity. Prices were processed by Ryna.
 #' @return prices in historical period
 #' @import dplyr
 #' @importFrom readr read_csv
-#' @importFrom tidyr gather
-#' @author KVC October 2017
+#' @author KVC October 2018
 #' @export
 get_hindcast_prices <- function(){
   # Silence package checks
-  prod <- price <- year <- value <- price_fy <- uniqueJoinField <- na.omit <-
-    FAO_country <- GCAM_commodity <- item <- deflator <- NULL
-
-  # Read in mappings
-  agluCtry <- suppressMessages(read_csv(system.file("extdata", "./mappings/AGLU_ctry.csv", package = "gcamland"), skip = 3))
-  iso_GCAM_regID <- suppressMessages(read_csv(system.file("extdata", "./mappings/iso_GCAM_regID.csv", package = "gcamland"), skip = 3))
-  GCAM_region_names <- suppressMessages(read_csv(system.file("extdata", "./mappings/GCAM_region_names.csv", package = "gcamland"), skip = 3))
-  FAO_ag_items_PRODSTAT <- suppressMessages(read_csv(system.file("extdata", "./mappings/FAO_ag_items_PRODSTAT.csv", package = "gcamland"), skip = 3))
-
-  # Read prices & production (we'll weight prices by production when we can aggregate)
-  faoPrices <- suppressMessages(read_csv(system.file("extdata", "./hindcast-data/fao_prices.csv", package = "gcamland")))
-  faoProd <- suppressMessages(read_csv(system.file("extdata", "./hindcast-data/fao_prod.csv", package = "gcamland")))
```

Comment:
We have deleted the read for this file and several others, but only `gdp_deflator.csv` and `fao_prices.csv` were deleted from the repo.  Are the other files still needed elsewhere?  If not, we could save ourselves some space by removing them too.

Reply:
We are still using these files elsewhere to calculate yields.